roles/fedreg-agent/README.md

```diff
@@ -0,0 +1,57 @@
+Federation registry agent Role
+===============================
+
+Federation registry agent (`fedreg-agent`) role deploy and configure the deployers from [rciam-federation-registry-agent](https://github.com/rciam/rciam-federation-registry-agent/tree/devel).
+
+
+Requirements
+------------
+
+* Ansible `2.9.11`.
+* Target to Debian GNU/Linux 10 (buster) hosts.
```

Comment:
Is there anything Debian 10 specific that wouldn't work on 9?

Reply:
No, but to be honest, I only tested it in **Debian GNU/Linux 10 (buster)**.